n2y/errors.py

```diff
@@ -98,6 +98,9 @@ class APIErrorCode(str, Enum):
     This can occur when the time to respond to a request takes longer than 60 seconds,
     the maximum request timeout."""
 
+    GatewayTimeoutError = "gateway_timeout"
```

Comment:
Are there any other missing keys that we should add?

Reply:
Yes, I just added them. Nice Catch!